src/ui/components/DropdownMenu.tsx

```diff
@@ -1,20 +1,33 @@
 import { ChangeEvent, FC, useEffect, useState, useCallback } from "react";
-import { Form } from "react-bootstrap";
+import { Form, Toast, ToastContainer } from "react-bootstrap";
 
 type DropdownMenuProps = {
   onSelectFeatureConfigId: (featureId: string) => void;
 };
 
 const DropdownMenu: FC<DropdownMenuProps> = ({ onSelectFeatureConfigId }) => {
   const [featureConfigs, setFeatureConfigs] = useState([]);
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+  const [toastVariant, setToastVariant] = useState<"success" | "danger">(
+    "danger",
```

Comment:
Additionally, all these pieces of state are tied together. If you were to use this pattern, I would reccomend making a type `type Toast { message: string; variant: "success" | "danger"; }`; and using `useState<Toast | null>(null)` here.